### PR TITLE
fixes #11916

### LINKED ISSUE #11858: Box (livestock Vox Animal) unpolished, no death state messages
#### Description of issue

The actual 'Box' animal is not consistent, when it dies for whatever reason the states after are not consistent with other mobs in lacking a death message and emitting a individual noise upon butchery post death, in general it could do with a rework or a touch up
#### Difference between expected and actual behavior

Some kind of message to indicate that the box that i have been cloning has died, I IC was busy managing station hydroponics in the presumption that the box i cloned would survive, however it soon died. I thought that it would die with a message (there was none in the chat) and just give off the usual gibbing noise when it is butchered.

What did result is that the box triggers its death sprite without warning, and upon butchering emits a deathscream, which is nonsensical as it is already dead without fabricating up a reasonable conclusion.
#### Steps to reproduce

Obtain box cloning kit, set up on either the station or the nitrogen laden (more specifically the regular station would be a good place to test due to the dangerous oxygenated atmosphere) vox base and feed box meat into the cloner, a box baby should emerge and shortly die of a lack of No2 air after. No death messages should occur as in line with the issue and the box sprite should be overturned hooves up.

Then proceed to butcher the box with a kitchen knife/butchering implement and upon the final gibbing carve it will issue a deathscream despite it already being dead.

### LINKED ISSUE #11916: Vox pig(box) still grows in size even while it's dead
#### Description of issue

If a Vox pig dies, it continues to grow in size until it reaches prize porker size.
Every stage of growth updates the icon to be the living state as well, so they look like they're living even
#### Difference between expected and actual behavior

Dead pigs shouldn't grow, much less seemingly spring back to life when they do
#### Steps to reproduce

Grow pigs in O2 rich environment so they suffocate
Watch them still grow as you butcher them
#### Length of time in which bug has been known to occur

Since the pigs existed I guess
#### Server revision

713564a7711ee3d83f7a869ebd7ee786c64abf3a 
#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by Kurfursten**
fixes #11916
fixes #11858
- bugfix: Boxen no longer grow after death (also fixes a sprite issue with dead boxen)
- bugfix: Boxen now properly give different amounts of meat based on how fat they were.
- bugfix: Boxen now scream when killed, not when gibbed.

200% tested

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: No one-line if statements.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm
Comment (Kurfursten) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: reee

Comment (Kurfursten) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: > comments don't autoresolve anymore

Github turned into a trash heap

**Review by PJB3005 — APPROVED** [review]

**Comment by Kurfursten** [comment]
You have to get _maintainer approval_ to make changes to your own branch? Wow, that's disgusting. I'm glad I don't have time to contribute anymore, this would be seriously painful if I had a future contributing code.

It's the git equivalent of "your comment is pending approval by our moderation staff".

**Comment by PJB3005** [comment]
> You have to get maintainer approval to make changes to your own branch? 

What?

**Comment by Kurfursten** [comment]
I was wrong, I read the patchnotes for github later.

When you put in an approval I misinterpreted that purpose of what that approval does. It seems dumb to me since a maintainer has to merge the PR in the first place so adding an approval step seems to do literally nothing, but it's not what I thought it was so I don't really care.

**Comment by gbasood** [comment]
That would be super dumb
Especially considering your own fork of the code is the repo that your branch lives on
Literal open source genocide

**Pull request merged**
